### DISCUSSION TITLE
Would love to contribute. Where to begin?

### DISCUSSION THREAD

**Discussion by AkshatRamanathan** [xpf0000/FlyEnv#121] [Q&A] [answered]
Im a Vue developer. Want to contribute. Please help with some English documentation and help, actively seeking issues to support and contribute to open source

**Comment by xpf0000** [comment]
Thanks. I will write a detailed document.

**Reply by AkshatRamanathan** [reply]
Thank you. Will look forward to some english documentation and steps to work on. Also please raise any issues in english if possible. Will be easier for more people to contribute if possible

**Reply by AkshatRamanathan** [reply]
Hi @xpf0000, any update on the docs? I can begin somewhere if you point me in the right direction

**Reply by xpf0000** [reply, marked as answer]
Not completely finished yet. Some are listed in [DEV.md](https://github.com/xpf0000/PhpWebStudy/blob/master/DEV.md).  The project structure is currently being adjusted to accommodate multi-person collaboration. Currently, a lot of code is mixed together. Adding or modifying modules requires modifying many files. It is currently being improved.

**Reply by AkshatRamanathan** [reply]
Thank you for the response. I will look forward to contribute. Please let me know when I can start. Thanks again for your help

**Reply by xpf0000** [reply]
docs has updated. [Development Guide](https://github.com/xpf0000/PhpWebStudy/blob/master/DEV.md).  You can fork the repo and run/test in local.

**Reply by AkshatRamanathan** [reply]
Sounds good. Will start working immediately. Please ensure new issues created are in english and if possible translations for old issues is mentioned in comments. Thank you for the help. Will check the DEV.md guide out

**Comment by whoami13apt** [comment]
帮助推广项目 本人summer13top
https://blog.csdn.net/m0_69732192/article/details/145537150?sharetype=blogdetail&sharerId=145537150&sharerefer=PC&sharesource=m0_69732192&spm=1011.2480.3001.8118
尽快审核处理谢谢您
